pkg/provider/docker/config_test.go

```diff
@@ -73,6 +73,7 @@ func TestDefaultRule(t *testing.T) {
 							},
 						},
 					},
+					ServersTransports: map[string]*dynamic.ServersTransport{},
```

Comment:
Why does this need to be added, since the docker provider isn't changing?

Reply:
Because we add an empty map here: https://github.com/traefik/traefik/pull/7407/files/19358f39deb407d44ef57c225748d013cb9fb64f#diff-31a8d4ce48a7e34ab87a44f923992da67fd82158ed35fac9368fd407fa797965R26 per default. We could probably revert that and check for nil…

Reply:
ugh. Nah, whatever. if it's done like this for the other fields, let's keep the consistency.